src/freeze_api.js

```diff
@@ -242,10 +245,11 @@ function applyChanges(root, changes, incremental) {
     newRoot = opSet.getIn(['cache', OpSet.ROOT_ID])
     if (newRoot === root) {
       newRoot = Object.assign(Object.create({_conflicts: root._conflicts}), root)
+      Object.defineProperty(newRoot, '_objectId', { value: root._objectId })
       opSet = opSet.setIn(['cache', OpSet.ROOT_ID], newRoot)
     }
   } else {
-    [opSet, newRoot] = materialize(opSet)
+    ;[opSet, newRoot] = materialize(opSet)
```

Comment:
@EthanRBrown Why is this semicolon necessary? I'm aware of the rule that lines starting with a parenthesis need a leading semicolon, but I didn't know there was such a rule for square brackets.

Reply:
If you were to write this:

```js
foo()
[bar, baz] = quux()
```

It would be interpreted as:

```js
foo()[(bar, baz)] = quux()
```

Reply:
Got it, thanks!